docs/monitoring-usage.rst

```diff
@@ -278,3 +278,91 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Metrics
+---------------------------
+
+The Stackdriver Monitoring API can be used to write data points to custom metrics. Please refer to
+the documentation on `Custom Metrics`_ for more information.
+
+To write a data point to a custom metric, you must provide an instance of
+:class:`~gcloud.monitoring.metric.Metric` created with a metric type as well as values for
+the labels associated with that metric type. The metric type can either
+be created using the :class:`~gcloud.monitoring.metric.MetricDescriptor` class or be
```

Comment:
I'll try to explain why I suggested phrasing this the way I did.

It's confusing to read about creating metric descriptors in text about writing custom metric data. They don't really go together. You may have many application processes writing data for the same metric type, and even more over time, considering restarts. You would only create the metric descriptor once. Especially since they are a limited resource (you only get 500), you might prefer to manage them rather explicitly, in admin tools separate from the application code.